docs/guide/controllers.md

```diff
@@ -615,6 +619,10 @@ Query results are provided to the template in the `items` variable:
         "viewType": itemViewType
     })) }}
 {% endfor %}
+
+{% if isPaginationEnabled %}
```

Comment:
We must document that variable, as well as `pageParameter` below.

In addition, there are three extra parameters that can be passed to the `ez_render_field` twig function: `enablePagination`, `disablePagination` and `itemsPerPage`. You can find the documentation for them in https://github.com/ezsystems/ezplatform-query-fieldtype/blob/master/doc/howto/customize_rendering.md.